src/editmap.cpp

```diff
@@ -41,20 +41,16 @@
 #include <vector>
 
 #define dbg(x) DebugLog((DebugLevel)(x),D_GAME) << __FILE__ << ":" << __LINE__ << ": "
-#define maplim 132
-#define pinbounds(p) ( p.x >= 0 && p.x < maplim && p.y >= 0 && p.y < maplim)
+
+static constexpr tripoint editmap_limit_min( 0, 0, -OVERMAP_DEPTH );
```

Comment:
Editmap limits are the same as for the main `map` class because it uses such a map instance.

Reply:
That is kind of temporary transitional variables while I changing the code to use new functions. There are would be have several types of similar limits and I plan to put all of them to one place eventually.